### PR TITLE
[MIG] Migrated project detail report module.

### PR DESCRIPTION
**Pull request by Vrushali-SerpentCS**

### CODE REVIEW

**Comment by JayVora-SerpentCS** [comment]
@Swapnesh-SerpentCS please review

File: project_detail_report/__manifest__.py [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
     'name': 'Project Report',
-    'version': '12.0.1.0.0',
+    'version': '13.0.1.0.0',
     'category': 'Project Management',
-    'author': 'Serpent Consulting Services Pvt. Ltd.',
     'license': 'AGPL-3',
+    'summary': """Print Project Detail report with task
+                    list and task details.""",
+
+    # Author
```
Comment (Swapnesh-SerpentCS) [inline project_detail_report/__manifest__.py]: Remove

File: project_detail_report/__manifest__.py [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
+    'author': 'Serpent Consulting Services Pvt. Ltd.',
     'website': 'http://www.serpentcs.com',
-    'summary': """
-      Print Project Detail report with task list and task details.""",
     'maintainer': 'Serpent Consulting Services Pvt. Ltd.',
+
+    # Dependencies
```
Comment (Swapnesh-SerpentCS) [inline project_detail_report/__manifest__.py]: Remove

File: project_detail_report/__manifest__.py [hunk 3]
```diff
@@ -1,21 +1,29 @@
...
     'depends': [
         'hr_timesheet',
     ],
+    # Data
     'data': [
         'views/project_report.xml',
         'report/project_qweb_report.xml',
     ],
+
+    # Technical
```
Comment (Swapnesh-SerpentCS) [inline project_detail_report/__manifest__.py]: Remove

File: project_detail_report/__manifest__.py [hunk 4]
```diff
@@ -1,21 +1,29 @@
...
     'website': 'http://www.serpentcs.com',
-    'summary': """
-      Print Project Detail report with task list and task details.""",
     'maintainer': 'Serpent Consulting Services Pvt. Ltd.',
+
+    # Dependencies
     'depends': [
         'hr_timesheet',
     ],
+    # Data
```
Comment (Swapnesh-SerpentCS) [inline project_detail_report/__manifest__.py]: Remove

File: project_detail_report/__manifest__.py [hunk 5]
```diff
@@ -1,21 +1,29 @@
 # See LICENSE file for full copyright and licensing details.
 
 {
+    # Module Information
```
Comment (Swapnesh-SerpentCS) [inline project_detail_report/__manifest__.py]: Remove

**Comment by Swapnesh-SerpentCS** [comment]
@JayVora-SerpentCS  Is it really necessary to add Non-technical things like this during migration?
I don't find it useful (It feels we are at beginner level to mention things like this)

**Comment by JayVora-SerpentCS** [comment]
> @JayVora-SerpentCS Is it really necessary to add Non-technical things like this during migration?
> I don't find it useful (It feels we are at beginner level to mention things like this)

Right but consider them as placeholders for now.

**Comment by JayVora-SerpentCS** [comment]
Review on tech grounds meanwhille

**Review by Swapnesh-SerpentCS — APPROVED** [review]

**Pull request merged**